### DISCUSSION TITLE
Where are the tables of the DB ?

### DISCUSSION THREAD

**Discussion by ttsoares** [vercel/community#2490] [Help] [answered]
### Summary

NextJS, Prisma, PostgreSQL.
Three days trying to follow the documentation with no avail...
Localy (npm build  && npm start) everything works fine, Have a local PG in my Linux system.
The deployment in Verced had the green light.
My .env.local was created by the Vercel CLI (vercel env pull .env.local).
The schema.prisma seems correct.
Did connect the DB with my deployed project.
But using the menus: Storage -> MY_DB_NAME the 'Browse the tables in this database'  still has no tables !!

### Additional information

When I try to write in the User table:

**Object { error: "\nInvalid `prisma.user.create()` invocation:\n\n\nThe table `public.User` does not exist in the current database.", user: null }**

Sure !   There is no table in the database...

### Example

https://vercel.com/ttsoares/prelo

**Comment by amyegan** [comment]
Sorry you're having trouble with your database, @ttsoares. It sounds like you're trying to use Vercel Postgres and following the steps in the [Getting Started with Vercel Postgres](https://vercel.com/docs/storage/vercel-postgres/quickstart#) documentation. Since you're using Prisma, I suspect you also followed the steps in the [Using an ORM to access your Postgres database: Prisma](https://vercel.com/docs/storage/vercel-postgres/using-an-orm#prisma) documentation. Please let me know whether that is correct or I misunderstood you.

I tried to reproduce the issue using the [Vercel Postgres + Prisma Next.js Starter](https://vercel.com/templates/next.js/postgres-prisma) as an example, but the starter schema and additional tables I added are all working normally. Do you have an example or public repo you can share so I can explore what's going wrong?

**Reply by ttsoares** [reply]
Thank you for a such fast reply !
This is the code I'm trying to deploy:
[https://github.com/ttsoares/prelo](https://github.com/ttsoares/prelo)
The code is ugly yet as I'm just trying to show a prototype of the tool to some friends.

**Reply by ttsoares** [reply, marked as answer]
It is working !!!!

My schema was missing the line: 

**previewFeatures = ["jsonProtocol"]**

Everything else was OK...

### Thank you very much !!  :-)

**Comment by ttsoares** [comment]
I was doing copy/paste with all configs... Can not understand how missed that one line at schema.prisma !!
Anyhow, there was no error message giving any hint about the issue.
Thanks a lot to this community and all those fine tools at disposal...

**Comment by ttsoares** [comment]
![err500](https://github.com/vercel/community/assets/42324195/ef36bbe8-d63f-43db-883b-6745a5907b9c)
Some weird started to happen again !
The application build and run locally and this using the DB (Postgresql) in
Vercel...
But the same exact application deployed do not run in Vercel as it seems
unable to reach the DB !!
I'm using the same code that was working May 17 but stopped now.
The only clue I have is that Prisma released an updated version yesterday
and I did apply it on my local system  :-(

Em qua., 17 de mai. de 2023 às 18:36, Amy ***@***.***>
escreveu:

**Comment by ttsoares** [comment]
Today I started from scratch to recreate this project from 'npm init'. Never touched my local Postgres, did all configurations pointing to the DB in Vercel. The application runs LOCALLY using the Postgres in VERCEL. 
**After the deployment, the SAME application is unable to talk with the Postgres IN Vercel !!!**

**Comment by ttsoares** [comment]
This is the final and definitive solution for this problem:
[https://www.prisma.io/docs/guides/other/troubleshooting-orm/help-articles/vercel-caching-issue](https://www.prisma.io/docs/guides/other/troubleshooting-orm/help-articles/vercel-caching-issue)
